compiler/ballerina-parser/src/test/resources/expressions/func-call/func_call_assert_11.json

```diff
@@ -58,52 +58,47 @@
+                                                                        {
+                                                                            "kind": "OPEN_PAREN_TOKEN"
+                                                                        },
+                                                                        {
+                                                                            "kind": "LIST",
+                                                                            "children": []
+                                                                        },
+                                                                        {
+                                                                            "kind": "CLOSE_PAREN_TOKEN"
+                                                                        }
                                                                     ]
-                                                                },
-                                                                {
-                                                                    "kind": "CLOSE_PAREN_TOKEN"
                                                                 }
                                                             ]
+                                                        },
+                                                        {
+                                                            "kind": "ELLIPSIS_TOKEN"
+                                                        },
+                                                        {
+                                                            "kind": "IDENTIFIER_TOKEN",
+                                                            "value": "subjects"
                                                         }
                                                     ]
                                                 }
                                             ]
-                                        },
-                                        {
-                                            "kind": "REST_ARG",
-                                            "children": [
-                                                {
-                                                    "kind": "COMMA_TOKEN",
-                                                    "isMissing": true
-                                                },
-                                                {
-                                                    "kind": "ELLIPSIS_TOKEN"
-                                                },
-                                                {
-                                                    "kind": "IDENTIFIER_TOKEN",
-                                                    "value": "subjects"
-                                                }
-                                            ]
```

Comment:
With `...` rage operator var-arg in  `foo( bar(5, a, age  baz() ...subjects), 5, a, age = 18, ...subjects)` is now recovered as range expression.

Reply:
should be fine